arcade/arcade/cli/main.py

```diff
@@ -499,3 +499,24 @@ def actorup(
         error_message = f"❌ Failed to start Arcade Actor: {escape(str(e))}"
         console.print(error_message, style="bold red")
         typer.Exit(code=1)
+
+
+@cli.command(
+    help="Create docs for a locally installed toolkit package", rich_help_panel="Tool Development"
+)
+def docs(
```

Comment:
Still thinking about prepending `toolkit-` to toolkit specific CLI commands @nbarbettini ? Now would be the time. lmk